libs/hdf-converters/src/converters-from-hdf/asff/transformers.ts

```diff
@@ -388,7 +388,12 @@ function createProfileInfoFindingFields(hdf: ExecJSON.Execution): string[] {
     targets.forEach((target) => {
       const value = _.get(profile, target);
       if (typeof value === 'string') {
-        typesArr.push(`${profile.name}/${target}/${value}`);
+        typesArr.push(
+          `${profile.name}/${target}/${value.replace(
```

Comment:
Do you think we should optimize for the general case and preemptively do this replacement on the profile name and target as well?